src/main/resources/application.properties

```diff
@@ -1,3 +1,2 @@
 spring.config.location=classpath:/
 @environment-properties@
-spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQL9Dialect
```

Comment:
again not sure if we should commit this since it could potentially interfere with other students' deployments